### PR TITLE
Casper customize

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .gitignore
Comment (banuragaxioned) [inline .gitignore:1]: Why remove this?

File: assets/css/screen.css [hunk 1]
```diff
@@ -42,6 +42,8 @@ production stylesheet in assets/built/screen.css
     --color-border: #e1e1e1;
     --color-wash: #e5eff5;
     --color-darkmode: #151719;
+    --color-hover:#e31c79;
```
Comment (banuragaxioned) [inline assets/css/screen.css:45]: let's remove both of these changes. line 45/46.

File: assets/css/screen.css [hunk 2]
```diff
@@ -3034,3 +3035,65 @@ file to add some of your own styles. Well, you've come to
...
+.fluid-width-video-wrapper{
+    padding-top: 382.2px !important;
+  }  
+
+.fluid-width-video-wrapper iframe,.fluid-width-video-wrapper iframe[name="fitvid0"]{
+    width: 100% !important;
+  }
+
+/* dark mode  */
+html.dark-mode  #gh-head,html.dark-mode  .site-footer, html.dark-mode .gh-dropdown { background: var(--color-header-darkmode);}
```
Comment (banuragaxioned) [inline assets/css/screen.css:3085]: let's remove all custom dark mode stuff

File: assets/css/screen.css [hunk 3]
```diff
@@ -3034,3 +3035,65 @@ file to add some of your own styles. Well, you've come to
...
+    background: var(--color-hover);
+}
+
+.gh-head-open #gh-head .gh-head-actions { background: transparent;}
+
+.post-card-image { aspect-ratio: 3/2;}
+
+.site-footer ,.gh-dropdown{ background: var(--ghost-accent-color);}
+
+.fluid-width-video-wrapper{
```
Comment (banuragaxioned) [inline assets/css/screen.css:3076]: why are these .fluid-width-video on here?
what's issue in the existing theme is it solving?

File: default.hbs [hunk 4]
```diff
@@ -81,10 +78,9 @@
     <footer class="site-footer outer">
         <div class="inner">
             <section class="copyright"><a href="{{@site.url}}">{{@site.title}}</a> &copy; {{date format="YYYY"}}</section>
-            <nav class="site-footer-nav">
+            {{!-- <nav class="site-footer-nav">
```
Comment (banuragaxioned) [inline default.hbs:81]: Can we add these back?

File: package-lock.json
Comment (banuragaxioned) [inline package-lock.json:1]: There are 2 lock files. Please remove package-lock.json as yarn is used upstream.

**Review by banuragaxioned — CHANGES_REQUESTED** [review]

File: assets/css/screen.css (hunk 3 above)
Comment (ghost) [reply assets/css/screen.css:3076]: This fix is for when we embed youtube video, on mobile devices the video overflows

**Pull request closed without merging**